VimeoNetworking/VimeoNetworking/VimeoClient.swift

```diff
@@ -388,6 +387,8 @@ final public class VimeoClient
         }
         catch let error
         {
+            self.responseCache.removeResponseForRequest(request)
```

Comment:
Do you think we'd ever _not_ need to remove the cached response when we have an error? From here it looks like regardless of what the error actually is we'll just dump the cache for this request. Or is that fine?